packages/api-nest/src/queue/services/twitter-extractor.service.ts

```
@@ -0,0 +1,281 @@
+@Injectable()
+export class TwitterExtractorService {
+  private readonly logger = new Logger(TwitterExtractorService.name)
+  private readonly SYNDICATION_API_URL = 'https://cdn.syndication.twimg.com/tweet-result'
+
+  /**
+   * Extract a Twitter thread
+   *
+   * @param url - Twitter/X URL
+   * @returns Thread extraction result
+   * @throws Error if tweet cannot be accessed
+   */
```

ESLint

[error] Missing JSDoc return type.

(valid-jsdoc)